src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2588,13 +2588,10 @@ Get\ more\ themes...=Get more themes...
 
 Add\ selected\ entries\ to\ database=Add selected entries to database
 The\ selected\ entry\ doesn't\ have\ a\ DOI\ linked\ to\ it.\ Lookup\ a\ DOI\ and\ try\ again.=The selected entry doesn't have a DOI linked to it. Lookup a DOI and try again.
-Cited\ By=Cited By
-Citing=Citing
 No\ articles\ found=No articles found
```

Comment:
"articles" is not a good term here. "papers" are no "articles" (because articles typically refer to journal articles). Use "publications"